docker/k8s-sglang-service.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: node.k8s.io/v1
```

Comment:
May we add health check, image pull policy and network policy?

Reply:
Sorry I've just starting to learn k8s for 4 days, after searching I'm still have no ideal about what network policy should do here.

Health check and pull policy was added.

Reply:
It's basically done, and I currently don't have the appropriate environment. It might need to wait for another reviewer to do a simple local verification. Thank you for your contribution!

Reply:
cc @merrymercy @Ying1123